src/api/general-services/__mocks__/get-work-results.js

```diff
@@ -0,0 +1,17 @@
+const { UnauthorizedError } = require('../../../utils/responses');
+
+const mockGetWorkResults = jest.fn((experimentId) => new Promise((resolve, reject) => {
+  const response = {
+    signedUrl: 'https://apiurl:port/blabla/asd/asd',
+  };
+  if (experimentId === 'unauthorizedExperimentId') {
+    const err = new UnauthorizedError('User does not have permissions to get worker results for this experiment');
+    err.status = 403;
```

Comment:
The `UnauthorizedError` should already have a property `error` set to 403 right? So you don't need to actually set it manually again?